src/raps/execute.ts

```diff
@@ -125,7 +131,18 @@ export const walletExecuteRap = async (
   type: RapTypes,
   parameters: RapSwapActionParameters<'swap' | 'crosschainSwap' | 'claimBridge'>
 ): Promise<{ nonce: number | undefined; errorMessage: string | null }> => {
-  const rap: Rap = await createSwapRapByType(type, parameters);
+  // NOTE: We don't care to track claimBridge raps
+  const rap =
```

Comment:
`createSwapByRap` type can also throw errors, so these will be swallowed up and will throw a different error when it tries to read `{ actions }` from `undefined`